src/Server/BaseServer.ts

```diff
@@ -180,6 +183,9 @@ export abstract class BaseServer implements IServer {
       const textFile = this.textFiles.get(path);
       if (!textFile) return { res: false, msg: `Text file ${path} not found.` };
       this.textFiles.delete(path);
+      //dispose of editor model
+      const model = getModel(this.hostname, path);
+      if (model) model.dispose();
```

Comment:
If you want to avoid a multi-line if, I'd do
```suggestion
      getModel(this.hostname, path)?.dispose();
```